app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Users list appears here</h1>
\ No newline at end of file
+<main class="container">
+  <% @users.each do |user| %>
```

Comment:
- [ ] [OPTIONAL] You have done a good job here and I would also like to add that it would be better to use ```<%= render @users %>``` instead of `@object.each`.